### DISCUSSION TITLE
Provide one free subdomain

### DISCUSSION THREAD

**Discussion by Timmi-Urge** [simple-login/app#1596] [Feature Requests]
Due to the recent drastic reduction in the number of free domains available to the free user, it has become extremely inconvenient to use the service. Yes, you have allocated a new domain (thank you for that), but it's still not enough. I ask you to provide ONE free subdomain for free users.

As an alternative to providing a free subdomain, I suggest adding another free domain. This will completely eliminate all inconveniences

**Comment by nguyenkims** [comment]
We have temporarily limited access on [simplelogin.com](http://simplelogin.com/) and [simplelogin.fr](http://simplelogin.fr/) as there is an increasing number of abuses on these domains and we are working on the best way to address this. As these domains are branded, a website can wrongly block all SimpleLogin domains at once which is detrimental to other domains.

In the meantime, we have added a new domain for the free plan and will probably continue doing so in the coming days.

Please rest assured that every domain has the same technical setup on SimpleLogin and we make sure every domain works everywhere.

If you have any idea on good domains that we should buy, please ping us on hi[at]simplelogin.io. A good domain is one with popular TLD (.com, .net, .io) and should preferably end with "mail" and has some connections with SimpleLogin or privacy in general.

**Comment by THEOCKID** [comment]
(Trying to minimize my irritation here)
Dear @Timmi-Urge ,
How about you PAY for the service?  Would you work for free?  Well, these find people don't either.  Sorry, but it burns my nads when people whine and moan about not getting enough for free.  Developers need to eat too! And shelter!

**Comment by danielbieli** [comment]
Have tested SSO wit hKeyCloak SingIn ?
